src/OrchardCore.Modules/OrchardCore.Users/Views/UsersAdminListSearch.cshtml

```diff
@@ -3,32 +3,37 @@
 @inject IDisplayManager<UserIndexOptions> DisplayManager
 @{
     var syntaxThumbnail = await DisplayManager.BuildDisplayAsync(Model, null, "Thumbnail");
+    var dropdownClassList = new List<string>()
+    {
+        "dropdown-menu"
+    };
+
+    if (CultureInfo.CurrentUICulture.IsRightToLeft())
+    {
+        dropdownClassList.Add("dropdown-menu-start");
+    }
```

Comment:
I'm not sure if it is better that way. It was one line before.

Reply:
this way we don't have formatting issues. Visual Studio formats the inline code in a confusing way so this way it's easier to read the code and VS does not have to reformat the view.